### DISCUSSION TITLE
Contacting Gegno issue [**ALL SPOILERS!**]

### DISCUSSION THREAD

**Discussion by esksp** [endless-sky/endless-sky#10769] [Q&A] [answered]
CAUTION - SPOILERS!

CAUTION - SPOILERS!

CAUTION - SPOILERS!

I've saved (did I?) that Gegno person on the uninhabited planet and brought him back home, was subsequently sent to the Quarg, and am now traveling back through Gegno space, when a mysterious device is supposed to show me a map marking a couple planets.
I expected those planets would show with a mission flag, as usually, but no mission appears and nothing hints as to which planets I'm supposed to look for.
I am not supposed to search all planets in the vicinity, am I? It's a big galaxy...

Is this a bug or did I miss something supposed to be obvious?

**Comment by Saugia** [comment, marked as answer]
In the dialog shown after your visit with the Scin in Ilirco, it states "showed markings for a few other planets throughout Gegno space", so not the entire galaxy, just the planets in there. I don't want to add indicators because these missions are purely meant to be stumbled upon; one of them happens automatically if you fly to civilian space, another is simply entering a system, and the rest are on the various inhabited planets in Gegno space. The device, at this point, has stopped working, so it wouldn't show the locations directly at this point.

The chances of the player running into at least 3 of these encounters is pretty good, and that's the minimum requirement for continuing the story. The concept of early Gegno missions is you really have to stick your nose out, because the Gegno don't want anything to do with you, as opposed to other aliens that have easy interaction and access after a few short conversations. You're not required to meet every single one, and technically, finding *all* of them prior to a certain point in the next mission means the Gegno become more worried about you because of how much you keep showing up, so I also wouldn't want to mark them and have the player always get that version of the dialog.

**Reply by esksp** [reply]
> throughout Gegno space", so not the entire galaxy

Still too vague. There are a lot of planets in Gegno space, and you don't even know if you're supposed to try any of those marked "restricted" (it has worked in other situations). Also, I had already landed before on some of the uninhabited, so it's very hard to keep tabs which I had already tried and which not. Not fun.

> I don't want to add indicators because these missions are purely meant to be stumbled upon

OK, I respect that, but in this case please rephrase the message, because the player _expects_ some indicators (something about the verb "marked"?).
I don't know, something like "message hints there might be systems" or some such, but avoid telling the player there was a "map with planets marked"...

> that's the minimum requirement for continuing the story

That's developer knowledge. At this point in the story the player is trying to get a contact with the Gegno guys, which (s)he understands is very difficult, so (s)he is _very_ wary of missing something.
The only way to alleviate this is to somehow _hint_ there is no need to search further, the player seems to have done enough already.
It is frustrating not knowing what has gone wrong, or even _if_ something has gone wrong. Why did I post this thread? Because I had some built-up understanding about how this game works, and all of a sudden I was left high and dry...)

PS:
Anyway, now I know a little more, and after having indeed sifted through the Gegno space and having found several encounters, I'm wondering what this message was supposed to be good for inside the story (except for chasing the player around)?
**Why on earth would the Gegno give you such a message?** 🤔

**Reply by danaris** [reply]
> In the dialog shown after your visit with the Scin in Ilirco, it states "showed markings for a few other planets throughout Gegno space", so not the entire galaxy, just the planets in there. I don't want to add indicators because these missions are purely meant to be stumbled upon; one of them happens automatically if you fly to civilian space, another is simply entering a system, and the rest are on the various inhabited planets in Gegno space. The device, at this point, has stopped working, so it wouldn't show the locations directly at this point.

Might I suggest adding something simple along the lines of not being entirely sure which planets are being indicated? That would convey that they're deliberately not being marked, and if you want to find them, you're going to have to poke around a bit.

**Reply by esksp** [reply]
> not being entirely sure which planets are being indicated?

That would be a good solution!

**Reply by Saugia** [reply]
>Anyway, now I know a little more, and after having indeed sifted through the Gegno space and having found several encounters, I'm wondering what this message was supposed to be good for inside the story (except for chasing the player around)?
Why on earth would the Gegno give you such a message? 🤔

The Scin are the sneaky, sly, smart type. They threw that device on your ship to get you to interact with them in Ilirco, thought the player doesn't know that yet. Some of them would be interested in you as an outsider, despite the Gegno overall (including many other Scin) being against it. Some Scin spend their lives dedicated to research to help their civilization - you're technically fair grounds to a few of them willing to go that far. This of course would be something picked up on/explained later once you are on good terms and are able to communicate with the Gegno.

I added this message so that there is at least *some* direction for the player here. Originally this "nudge" didn't exist, and I entirely relied on the player going around and visiting the various uninhabited planets by their own choice. However, feedback that I agreed with was that the player really doesn't have reason to revisit them, especially after first landing on them in the area prior to this part in the intro, therefore I added a way that pokes the player to go back.

Sure, I could adjust the description. I could also clarify "uninhabited" planets, to avoid the confusion with restricted ones. Though I'm not sure how to balance that, with "unsure which planets are being indicated", but then have it clearly indicate uninhabited planets only.

**Reply by danaris** [reply]
"It's not entirely clear which particular planets are being indicated, aside from the fact that they are all currently uninhabited."

(How does it indicate that? This can be left to the imagination!)

**Reply by Saugia** [reply]
I'd imagine some people may ask, "how are you interpreting/is the device displaying they're specifically uninhabited?". Though maybe we just say, for the sake of avoiding specifics, we leave it at that anyways (other places in the game intentionally avoid rabbit holes like that).

Yes, maybe just leave it to imagination? The more I try and explain the device, the more questions about Gegno tech and what specific cognitive abilities the player has are added.

**Reply by danaris** [reply]
They use the universal symbol for "uninhabited planet", just like the universal credit system and universal ship controls ^_^

**Reply by Saugia** [reply]
Also,

>The only way to alleviate this is to somehow hint there is no need to search further, the player seems to have done enough already.
It is frustrating not knowing what has gone wrong, or even if something has gone wrong. Why did I post this thread? Because I had some built-up understanding about how this game works, and all of a sudden I was left high and dry...)

You do get a notification of sorts. If you've done at least 3, you get a message from the Quarg that leads to the next part. If you do all 6, or some combination of conditions, you get the same message (albeit a different dialog).

**Reply by Saugia** [reply]
Looking back at the mission, it does explicitly state what type of planets at least, rather than just "planets" in general; would just clarifying them as uninhabited be sufficient?

**Reply by esksp** [reply]
> to get you to interact with them in Ilirco

I didn't really get it: They scanned my palm, gestured and that was it. Quite limited as interactions go.


> I entirely relied on the player going around and visiting the various uninhabited planets by their own choice.

Which is indeed unlikely to happen, especially if you've already visited them before triggering the story line...

But given the player has to travel through different systems coming back from the Quarg (which are on the extreme East), can't you have pop-up messages interrupting travel, stating that having become suspicious of those supposedly uninhabited planets, you've decided to scan those you pass, and your scan just noticed an anomaly here? The first time you could also drop a hint there might be more, so the curious player will start exploring, even if (s)he has already explored those systems.
Once (s)he has triggered the message, the player will know (s)he needs to jump around the whole area looking for more, which is quickly done (= pop-up or no pop-up at system entry, you don't need to land).
That's assuming there is no way the player could cross the Gegno territory from East to West without passing at least one of those trigger planets.


> I'm not sure how to balance that, with "unsure which planets are being indicated", but then have it clearly indicate uninhabited planets only.

The "automatic scan" idea would kind of solve that because if there was no "scan" message there is no need to land. You'd only land if and when a pop-up tells you "you have spotted suspect activity on planet XYZ, you should investigate". 

> You do get a notification of sorts. If you've done at least 3, you get a message from the Quarg that leads to the next part.

Really? I didn't get one. I only saw the Quarg when I visited them, then I found (IIRC) 4 Gegno encounters, in any case more than 3, and nothing. What was this message supposed to look like and where would it happen? I eventually left for Syndicate space.

> would just clarifying them as uninhabited be sufficient?

I don't know about sufficient, but _definitely better_, since it would reduce the number of systems you have to search.

**Reply by Saugia** [reply]
I mispoke about the Quarg encounter; my apologies. Between 3 and 6, you have to re-land on the Quarg ringworld to get dialog. At 6, or a combination of conditions (mainly hostile ones, such as you attacking the Gegno factions during a duel, stealing the superheavy, etc.), the Quarg then call you back. So, in one case, you have to passively come across to the Quarg again. In another, and what's more likely since the player will explore more and the uninhabited planets, you will be pulled more aggressively, with the Gegno being frustrated you're still trying to stick your nose into their business. Both paths are equally valued.

So, if you did 3 missions, then you are at the minimum, and the Quarg will speak to you if you go to their ring. If you're at 6, or other conditions, the Quarg will beckon you.

I don't want to add messages or blips along the way for specific planets. I think "there might be other opportunities you can interact with the gegno" and referring to the uninhabited planets is more than enough to continue this. Something we avoid is holding the player's hand - with the Gegno story, I intentionally have made it so that you *have* to go out of your way to find encounters, not be told them or hinted to them in every case. I don't think landing on planets is that difficult of a way to do so.

Sure, I'll clarify to uninhabited then. But I believe directly pointing to places isn't necessary; the very first mission in the Gegno series is landing on the random uninhabited planet and stumbling into the Gegno; that sort of is to lead the expectation that this can happen in Gegno space in general.

**Reply by Saugia** [reply]
Also, with the message already indicating specifically what type of planets (moon, terrestrial, desert), that combined with just clarifying "uninhabited" I think should be good.

**Reply by esksp** [reply]
> Between 3 and 6, you have to re-land on the Quarg ringworld to get dialog. 

Uuuh, how is one supposed to guess _that_? There is definitely need for some dialogue rewriting here, so that players eventually get a clue of what there is to do.

Sorry, but it's a space combat/exploration game, not a puzzle where you'd gladly click on every pixel just in case it triggers something... :-D

> So, if you did 3 missions, then you are at the minimum, and the Quarg will speak to you if you go to their ring. If you're at 6, or other conditions, the Quarg will beckon you.

Except there was little (if any) chance I might get back to that Quarg ring anytime soon!
It's not like that system is on a main travel route, so why would I?...
As I said, needs some hinting or rewriting.

(BTW, since we're about unlikely conditions, why do you need to go back to _human_ space to trigger the **initial Bunrodea** mission?
At this point in the game I almost never land in human space anymore (except some times to refuel while traveling from Remnant to western Wanderer space). The initial Bunrodea mission should trigger on _any_ friendly system.)

> Something we avoid is holding the player's hand

There is some leeway between "holding hands" and "requiring a secret handshake"... :-D

> I intentionally have made it so that you _have_ to go out of your way to find encounters

But you also increase exponentially the risk that the player will decide there is nothing more to see here, and leave Gegno space definitively. Like it happened to me. :-(
Now you might say "too bad for them", but that's not how game difficulty works. Else just require typing a secret, impossible to find 12-character password to access advanced game content, and too bad for those who aren't able to guess it...  :-p

> I don't think landing on planets is that difficult of a way to do so.

Difficult no, but it's
1. that annoying "let's waste the player's time" feeling, where you have unneeded courier/taxi missions which don't bring nothing story wise, but are supposed to keep the player busy for an additional couple minutes. Those missions are OK when there is some story progression behind them, but very often it's just that there is a bunch of uninteresting and unneeded filler system hopping missions between "real mission n" and "real mission n+1", just to fill some additional time. 
2. The search will be enjoyable _if and when you know what you are doing and looking for_. It's utter drudgery when you don't  know what you're supposed to do at all, and just go through the motions ("What can I do in ES to initiate contact? Let's try this, then that, and if nothing works, use Internet search."). 


> Sure, I'll clarify to uninhabited then. But I believe directly pointing to places isn't necessary

Might be. The next players' feedback will tell if they successfully managed to guess what you expected them to do.

**Reply by Saugia** [reply]
>that annoying "let's waste the player's time" feeling, where you have unneeded courier/taxi missions which don't bring nothing story wise, but are supposed to keep the player busy for an additional couple minutes. 

I disagree. These missions don't waste the player's time - each one provide some lore insight to the area and possible knowledge going into the deeper part of the story later. You interact with individuals, traditions, etc., these small early leads that will lead into something later. Whether or not landing on a planet and it giving dialog is a waste of time is subjective. You shouldn't be looking for these to strictly progress the story; the missions themselves *are* part of the story.

>The search will be enjoyable if and when you know what you are doing and looking for.

Again, the device explicitly says a moon, a desert planet, and a terrestrial planet. Now that I have a PR open suggesting they're also uninhabited, there's not really anything else that can be done to guide to these worlds without just saying, "go here, this is where you should go" as a pop-up or arrow, which defeats the purpose of these being encounters you have to go out of your way for.

The Scin encounter is actually pretty in-depth for a new alien species, as they go out of their way to interact with you and there are many interaction options with them for an interesting experience. From the way you wrote about it: "I didn't really get it: They scanned my palm, gestured and that was it. Quite limited as interactions go.", you have your own opinion of how interesting the content is, which is of course fine.

So, after that mission, I still feel that being told specific planets and the general sense that going around and landing on uninhabited planets again will continue the story is more than enough of a guide to lead the player to the next part of the series. You only need to land on 3 of the planets - if you give up after 1 or 2 and leave Gegno space, I feel that's the player's decision. Also of course fine.

**Reply by Saugia** [reply]
I did just have a thought though. What I could do is add a mission that does not add any markers or waypoints, but floats in the mission log that simply says "Fly throughout Gegno space looking for encounters". This is done elsewhere, for example, with the Giftbringer, where you have to find it and there is no markers, but the mission title does sit in the log as an active mission until you do find it.

**Reply by esksp** [reply]
> I disagree. These missions don't waste the player's time - each one provide some lore insight to the area and possible knowledge going into the deeper part of the story later. 

We're not speaking about the same thing: I was talking about those general missions in which, to do be able to do something, you have to make a couple trips to _already well known_ planets to fetch/do something before being able to do what you already know has to be done. Like, where you have to fetch something from X and something from Y, both pretty far away, before being able to do the task in hand.
Note I don't mind them, but they don't bring anything into the story except making the whole thing last longer.


> Again, the device explicitly says a moon, a desert planet, and a terrestrial planet. Now that I have a PR open suggesting they're also uninhabited

I saw your PR and I think it will be helpful, because the "moon/desert/terrestrial" hint doesn't bring much useful information: This description can fit _any_ of those unknown planets, uninhabited or not, you'll only know it once you've visited them (how do I know one of those "restricted" planets doesn't fit the description?).
"Uninhabited" limits indeed that choice and prevents wasting time trying all and each "restricted" planet for some useful interaction.


> The Scin encounter is actually pretty in-depth for a new alien species, as they go out of their way to interact with you and there are many interaction options with them for an interesting experience. From the way you wrote about it: "I didn't really get it: They scanned my palm, gestured and that was it. Quite limited as interactions go.", you have your own opinion of how interesting the content is, which is of course fine.

My actual point was just that I don't know how to go further beyond that: It was an encounter, a first contact, but there is no clear suggestion what I can do after that. Maybe I need to wait for something to happen, maybe I need to do something specific, but I have no clue at all... So I tried a couple things, and when nothing worked, I just left. Nothing to do with "interest", just with frustration.


> You only need to land on 3 of the planets - if you give up after 1 or 2 and leave Gegno space, I feel that's the player's decision. Also of course fine.

But I _did visit 4_ of them, and didn't get any kind of clue there was something I should do.
Of course I could had kept visiting other planets to (unknowingly) hit that "6" limit you spoke of, but I was under the impression that I already had landed on all planets I was allowed to land on.
I was apparently wrong, but I only know this because we're talking about it here, which means it wasn't too difficult to miss.

**Seriously, I'm not badmouthing or belittling your work, I'm just saying that at some point you don't know how to continue, and after trying in vain some things you _think_ might work, your only option is to abandon and hope some unforeseen future event will unlock things.**
The player confronts the story through the game's tools (like hailing planets/ships, landing on planets, scanning stuff, etc.), so after trying the more obvious tools without _any_ result, you feel you're in a dead end. What I'm trying to say is it's missing a "cold--warmer--hot--you're burning" type feedback which would keep the player committed to keep trying--in the right direction.
Besides that, the Gegno _are_ interesting, that's why I spent hours trying and finally started this thread...

.

> What I could do is add a mission that does not add any markers or waypoints, but floats in the mission log that simply says "Fly throughout Gegno space looking for encounters". 

Good idea. 👍

**Reply by Saugia** [reply]
No worries here, I don't believe you're belittling my work at all. I appreciate your description of your experience; I've had many different responses to it. With having that mission floating in the mission log, the player will at least know that so long as it is there, there is something to find.

**Reply by esksp** [reply]
There is an issue I'd like to address though: **You said that after 3 of those "encounters" the player is supposed to go back to the Quarg ring**. No problem, but that definitely needs some hinting too, because as I said above there was no way I would had guessed that!...

The Gegno work a little too different from all the other ES aliens (I've met them all by now I think, and have a trunk full of licenses...).
Usually you have to be a little diplomatic to get a foot in the door, then show some goodwill and do some chores, and the story line starts.
At first sight the Gegno seem to work in a similar manner, except that what seems like those goodwill chores doesn't seem to do anything: I've collected asteroids for at least one Real Time hour (having to use a clumsy big heavy warship too!), and nothing happened except those missions getting more and more rare, forcing me to take off/land a dozen times to get the next one. **At this point I must assume those asteroid hunting jobs don't have anything to do with starting a Gegno story line, do they?**
If not, what has?   -scratching head-

**Reply by Saugia** [reply]
>There is an issue I'd like to address though: You said that after 3 of those "encounters" the player is supposed to go back to the Quarg ring. No problem, but that definitely needs some hinting too, because as I said above there was no way I would had guessed that!...

"Supposed to" was poor choice of words for me. It's more for a different encounter; the player can go to the Quarg ring sooner and get the one discussion, or they can continue to explore until the Quarg pull them in if they continue with more encounters. It's more to say, the Quarg said to be careful, and the second path (getting all encounters and the Gegno being upset) is actually the more intended path.

If I were to get a nudge or hint to visit Quarg, the player would almost always go do that first and the second path would almost never happen as opposed to the chance of them landing on the world being equivalent to the chance they get all 6 encounters.

>The Gegno work a little too different from all the other ES aliens

This was the intent. Fundamentally, they work the same - interact initially with the species, have some talks, they slowly open up planets. In the Gegno's case, you have to go out of your way to start the encounter - it is to add variety to the galaxy in that not every single alien faction will just accept you after a little diplomatic action. This is a case where one doesn't want anything to do with you to begin with.

>At first sight the Gegno seem to work in a similar manner, except that what seems like those goodwill chores doesn't seem to do anything: I've collected asteroids for at least one Real Time hour (having to use a clumsy big heavy warship too!), and nothing happened except those missions getting more and more rare, forcing me to take off/land a dozen times to get the next one. At this point I must assume those asteroid hunting jobs don't have anything to do with starting a Gegno story line, do they?
If not, what has? -scratching head-

This is simply because this is where current content stops. After a few mining missions, I have missions planned to pop up or offer you more and lead to events later on. The missions do count; it is similar to Coalition space where you need to do so many odd coalition jobs to progress their story.

**Reply by esksp** [reply]
> the player can go to the Quarg ring sooner and get the one discussion, or they can continue to explore until the Quarg pull them in if they continue with more encounters.

I get that, what I'm trying to say is that apparently it is easy to _not_ get all 6 encounters (after 4-5 I thought I had visited everything), which means the Quarg won't take any initiative, and given the Quarg system is located in a dead end with little to do, it's unlikely for the player to visit them anytime soon (maybe in search of informations about the Umbral Reach, but not necessarily so since they aren't too close, or too eager to share information).

There is room for streamlining. Maybe lower that "Quarg" trigger from 6 to 5 (or even 4) encounters?
A player who doesn't spend his days on GitHub would had given up, since there doesn't seem to be a way forward. The Gegno need to be difficult to reach, but not seem impossible, lest the player decides they are just no-story window dressing like the Archons (don't mean that pejoratively, it's just there is no player interaction with the Archons).


> it is to add variety to the galaxy in that not every single alien faction will just accept you after a little diplomatic action. 

That's very good, but you need to keep some challenge-reward balance, lest only obsessive-compulsive players manage to contact them (without hunting Internet for spoilers)!... :-D
 

> This is simply because this is where current content stops.

Yikes. I see.
Okay, I'll be waiting for some future version to further explore the Gegno...

> The missions do count; it is similar to Coalition space where you need to do so many odd coalition jobs to progress their story.

Yes, that's why I did over 30 of those utterly repetitive (and tedious with a bulky warship) asteroid mining missions before deciding it didn't lead anywhere.
